apps/web/app/pages/sbtc-rewards/components/acquire-sbtc-grid.tsx

```diff
@@ -0,0 +1,76 @@
+import { Box, Flex, GridProps, styled } from 'leather-styles/jsx';
+import { DummyIcon } from '~/components/dummy';
+import { BitcoinIcon } from '~/components/icons/bitcoin-icon';
+import { StacksIcon } from '~/components/icons/stacks-icon';
+import { AcquireSbtcGridLayout } from '~/pages/sbtc-rewards/components/acquire-sbtc-grid.layout';
+
+import { Button } from '@leather.io/ui';
+
+export function AcquireSbtcGrid(props: GridProps) {
+  return (
+    <AcquireSbtcGridLayout
+      primaryCell={
+        <Flex flex={1} flexDir="column" justifyContent="space-between" p="space.05">
```

Comment:
Maybe these slot props can be seperated out from here?

Reply:
Not sure I entirely follow this comment actually. Separate components?

Reply:
Yes I just mean not to pass in huge JSX chunks as params. They can be in this file or wherever